docs/content/open-api/openapi.yaml

```diff
@@ -0,0 +1,1296 @@
+                                  type: string
+                        body:
+                          description: A list of the terms in this rule
+                          type: array
+                          items:
+                            properties:
+                              index:
+                                description: The location of this term in the list (starts at 0)
+                                example: 1
+                                type: number
+                              terms:
+                                description: The type/value pairing for this term
+                                type: array
+                                items:
+                                  properties:
+                                    type:
+                                      description: The type of the term variable
+                                      example: var
+                                      type: string
+                                    value:
+                                      description: The list of types and values for the term variable
+                                      type: array
+                                      items:
+                                        properties:
+                                          type:
+                                            type: string
+                                          value:
+                                            type: string
+    200Metrics:
+      type: object
+      properties:
+        metrics:
+          type: object
+          properties:
+            timer_rego_input_parse_ns:
+              description: Time taken (in nanonseconds) to parse the input
+              type: number
+              example: 69994
+            timer_rego_query_parse_ns:
+              description: Time taken (in nanoseconds) to parse the query
+              type: number
+              example: 4096
+            timer_rego_query_compile_ns:
+              description: Time taken (in nanoseconds) to compile the query
+              type: number
+              example: 4096
+            timer_rego_query_eval_ns:
+              description: Time taken (in nanonseconds) to evaluate the query
+              type: number
+              example: 69994
+            timer_rego_module_parse_ns:
+              description: Time taken (in nanoseconds) to parse the input policy module
+              type: number
+              example: 12345
+            timer_rego_module_compile_ns:
+              description: Time taken (in nanonseconds) to compile the loaded policy modules
+              type: number
+              example: 69994
+            timer_server_handler_ns:
+              description: Time taken (in nanoseconds) to handle the API request
+              type: number
+              example: 631000
+            timer_server_read_bytes_ns:
+              description: '*Description is forthcoming*'
```

Comment:
Yeah, I'll check what this really measures.

Reply:
Where did all of these come from now again? Will see if I can track them down in the code.

Reply:
Some are from https://github.com/open-policy-agent/opa/blob/master/docs/content/rest-api.md#performance-metrics, two or three other metrics appear consistently in responses. The 'instrument=true only' responses, I've indicated in the 'description' field.